LogicalTypes.md

```diff
@@ -539,6 +544,26 @@ The sort order used for `INTERVAL` is undefined. When writing data, no min/max
 statistics should be saved for this type and if such non-compliant statistics
 are found during reading, they must be ignored.
 
+#### INTERVAL_YEAR_MONTH
+
+`INTERVAL_YEAR_MONTH` is used to represent a year-month time interval, such as
+`4 years and 6 months`. It must annotate an `int32` that stores the total number
+of months as a signed integer, which represents the interval and can be negative.
+The time duration is independent of any timezone.
+
+#### INTERVAL_DAY_TIME
+
+`INTERVAL_DAY_TIME` is used to represent a day-time time interval, such as
+`5 days, 10 hours and 30 minutes`. It must annotate a 16-byte `FIXED_LEN_BYTE_ARRAY`
+that stores the total number of nanoseconds representing the interval. The value is
```

Comment:
Why call it "day-time" if it only stores nanoseconds? Please let's make the naming clear and non-misleading.